testsuites/lbt-plutus/lbt-plutus-rust/Cargo.toml

```diff
@@ -4,10 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+plutus-ledger-api = { git = "https://github.com/mlabs-haskell/plutus-ledger-api-rust", features = [
+  "lbf",
+], ref = "szg251/fix-lbf", rev = "4460f2ff8af7993a24209ebe28df5ef9b3b3b397" }
```

Comment:
Yeah upgrades are going to be tricky.

Imagine the following!

Each `build.nix` file also declares a `packages.upgrade-script` which is actually a devShell in which `cargo update`, `npm i`, `spago2nix generate` and sh happens?

Hmm, curious if that could be smtn useful

Reply:
I'll create an issue to solve the updating process. This needs some thinking